compose.yaml

```diff
@@ -6,23 +6,13 @@ services:
       - database
     restart: unless-stopped
     ports:
-      # HTTP
-      - target: 8080
-        published: ${HTTP_PORT:-8080}
-        protocol: tcp
-      # HTTPS
-      - target: 443
-        published: ${HTTPS_PORT:-443}
-        protocol: tcp
-      # HTTP/3
-      - target: 443
-        published: ${HTTP3_PORT:-443}
-        protocol: udp
+      - ${API_PORT:-8080}:8080
 
   web:
     build: ./web
     ports: 
-      - 3000:3000
+     - ${WEB_PORT:-8080}::3000
```

Comment:
there's a double colon here, I think you need only one

Reply:
Thanks you !